recipe/cross_compile_support.sh

```diff
@@ -15,19 +19,52 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
     fi
     if [[ "${BUILD_PLATFORM}" == "linux-64" && "${HOST_PLATFORM}" == linux-* ]]; then
         mamba create -n sysroot_${HOST_PLATFORM} --yes --quiet sysroot_${HOST_PLATFORM}
+        HOST_PLATFORM_ARCH=${HOST_PLATFORM:6}
         if [[ -f ${CI_SUPPORT}/../recipe/yum_requirements.txt ]]; then
             for pkg in $(cat ${CI_SUPPORT}/../recipe/yum_requirements.txt); do
                 if [[ "${pkg}" != "#"* && "${pkg}" != "" ]]; then
-                    mamba install "${pkg}-cos7-${HOST_PLATFORM:6}" -n sysroot_${HOST_PLATFORM} --yes --quiet || true
+                    mamba install "${pkg}-cos7-${HOST_PLATFORM_ARCH}" -n sysroot_${HOST_PLATFORM} --yes --quiet || true
                 fi
             done
         fi
         export QEMU_LD_PREFIX=$(find ${CONDA_PREFIX}/envs/sysroot_${HOST_PLATFORM} -name sysroot | head -1)
         if [ -f ${CI_SUPPORT}/${CONFIG}.yaml ]; then
             echo "CMAKE_CROSSCOMPILING_EMULATOR: " >> ${CI_SUPPORT}/${CONFIG}.yaml
-            echo "- /usr/bin/qemu-$(echo $HOST_PLATFORM | cut -b 7-)-static"  >> ${CI_SUPPORT}/${CONFIG}.yaml
+            echo "- /usr/bin/qemu-${HOST_PLATFORM_ARCH}-static"  >> ${CI_SUPPORT}/${CONFIG}.yaml
             echo "CROSSCOMPILING_EMULATOR: " >> ${CI_SUPPORT}/${CONFIG}.yaml
-            echo "- /usr/bin/qemu-$(echo $HOST_PLATFORM | cut -b 7-)-static"  >> ${CI_SUPPORT}/${CONFIG}.yaml
+            echo "- /usr/bin/qemu-${HOST_PLATFORM_ARCH}-static"  >> ${CI_SUPPORT}/${CONFIG}.yaml
+        fi
+
+
+        if [[ "${CUDA_COMPILER_VERSION}" == "11.2" ]] && [[ "${CDT_NAME}" == "cos8" || "${CDT_NAME}" == "cos7" ]]; then
+            # We use cdt_name=cos7 for rhel8 based nvcc till we figure out
+            # a stable cos8 replacement.
+            EXTRACT_DIR=$(mktemp -d)
+            pushd ${EXTRACT_DIR}
+                if [[ "${HOST_PLATFORM_ARCH}" == "aarch64" ]]; then
+                    CUDA_HOST_PLATFORM_ARCH=sbsa
+                else
+                    CUDA_HOST_PLATFORM_ARCH=${HOST_PLATFORM_ARCH}
+                fi
+                curl -L https://developer.download.nvidia.com/compute/cuda/repos/rhel8/sbsa/version_11.2.2.json > versions.json
```

Comment:
Should we use `CUDA_HOST_PLATFORM_ARCH` instead of hardcoding to `sbsa` for the URL here? I imagine the versions are consistent but I'd rather be safe than sorry.

Reply:
I think that should be possible.

Reply:
Done in https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/210/commits/a7d3bda1722c2805a05f801f96fe4f06f673de07